utils/cn.ts

```diff
@@ -0,0 +1,6 @@
+import { ClassValue, clsx } from "clsx";
```

Comment:
We already have `lib/utils.ts` file that has the same function `cn`
Can we remove this one?